tensorboard/components/tf_imports/polymer-externs.html

```diff
@@ -0,0 +1,4 @@
+<script src="externs/closure-types.js"></script>
+<script src="externs/polymer-externs.js"></script>
+<script src="externs/polymer-internal-shared-types.js"></script>
```

Comment:
This file doesn’t have an `@externs`. Do we need to do anything special
to handle it correctly during vulcanization?

Reply:
There seems to be a discrepancy with the v3 source which internal polymer is synced with. As per your offline feedback, I decided to use the `jscomp-externs` attribute to detect an extern instead of looking for `@externs` in the code.